Description: Upstream changes introduced in version 4.8.30-1stormos1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 db4.8 (4.8.30-1stormos1) unstable; urgency=low
 .
   * Package for StormOS.
 .
 The person named in the Author field signed this changelog entry.
Author: Andrew Stormont (astormont) <andyjstormont@googlemail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- db4.8-4.8.30.orig/test/scr030/chk.build
+++ db4.8-4.8.30/test/scr030/chk.build
@@ -118,7 +118,3 @@ done
 r
 r "--disable-static"
 r "--enable-cxx"
-r "--enable-java"
-r "--with-tcl=/usr/local/lib/tcl8.4"
-r "--enable-test --with-tcl=/usr/local/lib/tcl8.4"
-r "--enable-cxx --enable-java --with-tcl=/usr/local/lib/tcl8.4"
